src/VisualStudio/Core/Def/Implementation/InheritanceMargin/MarginGlyph/TargetMenuItemViewModel.cs

```diff
@@ -13,28 +13,56 @@ namespace Microsoft.VisualStudio.LanguageServices.Implementation.InheritanceMarg
     /// <summary>
     /// View model used to show the MenuItem for inheritance target.
     /// </summary>
-    internal class TargetMenuItemViewModel : InheritanceContextMenuItemViewModel
+    internal class TargetMenuItemViewModel : InheritanceMenuItemViewModel
     {
+        /// <summary>
+        /// The margin for the default case.
+        /// </summary>
+        private static Thickness s_defaultMargin = new Thickness(4, 1, 4, 1);
+
+        /// <summary>
+        /// The margin used when this target item needs to be indented when the target is shown with the header.
+        /// e.g.
+        /// 'I↓ Implemented members'
+        ///       Method 'Bar'
+        /// 'I↑ Implementing members'
+        ///       Method 'Foo'
+        /// It is 20 because the default left margin is 4, and the size of the image moniker(e.g. I↑) in the header is 16.
+        /// </summary>
+        private static Thickness s_indentMargin = new Thickness(20, 1, 4, 1);
```

Comment:
slightly worried about these constants.  have you tested VS under different resolutions and DPI settings?  Similarly, we might want to see what this looks like for other localizations.

Reply:
I did this simply because I saw node in  solution explorer is expanded, the nested item was aligned next to the parent's image.
![image](https://user-images.githubusercontent.com/24360909/115595706-1bc5c880-a28c-11eb-8413-921d59131019.png)
I'll try to see if there is any VS requirement for indentation.

Reply:
My point isn't that indentation is bad. My point is that i don't trust that what you did above will work properly in all resolutions/dpis/locals.

Reply:
I didn't find any useful UX guide for indentation,
however, there are two way that we could do here
1. By default the WPF tree view is indented 19 per level (depends on the expander size, if it is larger it might be larger) https://docs.microsoft.com/en-us/dotnet/desktop/wpf/controls/treeview-styles-and-templates?view=netframeworkdesktop-4.8
2. In Solution Explorer, the indentation is 18 per level, and the expander size is also 16 * 16.  So I feel 18 + 4 = 22 would be a good value for us.

Reply:
all i'm saying is:

I want validation that this looks ok in different resolution/dpis/text-editor-zoom level.  Please confirm the following changes still look good:

![image](https://user-images.githubusercontent.com/4564579/115604899-bb885400-a296-11eb-8bc2-b0c984f8bc17.png)


![image](https://user-images.githubusercontent.com/4564579/115604906-be834480-a296-11eb-8494-bebd6df5c3c9.png)


![image](https://user-images.githubusercontent.com/4564579/115604917-c4792580-a296-11eb-9a09-403e6ff3a08c.png)

If this all looks ok, then *awesome*.  We just need to do due diligence here.

Reply:
200% scale (windows settings) at 4k
![image](https://user-images.githubusercontent.com/24360909/115626598-96a1da00-a2b2-11eb-91df-328bb1c43c77.png)

350% scale (windows settings) at 4k
![image](https://user-images.githubusercontent.com/24360909/115627074-41b29380-a2b3-11eb-99d1-b8942afcdec3.png)



400% scale in VS
![image](https://user-images.githubusercontent.com/24360909/115626738-c355f180-a2b2-11eb-83a7-495039019ceb.png)

200% scale in VS
![image](https://user-images.githubusercontent.com/24360909/115626829-e41e4700-a2b2-11eb-933e-d31630eb739c.png)

100% scale in VS
![image](https://user-images.githubusercontent.com/24360909/115626957-13cd4f00-a2b3-11eb-8eb3-fc16edf7ea8e.png)

Reply:
VS guidelines are at [this internal link](https://devdivdesignguide.azurewebsites.net/processes/craftsmanship-checklist/#resolution-and-screen-size)